### DISCUSSION TITLE
FROGBLOCK license

### DISCUSSION THREAD

**Discussion by Polyducks** [replit/kaboom#570] [General]
Hello,

I'm the author of the FROGBLOCK font. 

Hosting the font on Github is really pushing the good will of the license - but please at the very least abide by the part of the license which requires you to credit it. The license states that usage of the font in an editor must make it clear that Polyducks is the author of the font. Please rename 'FROGBLOCK.ttf' to "FROGBLOCK-by-Polyducks.ttf", "Polyducks-FROGBLOCK" or something similar. If this is not possible, please remove my font from your project and repository.

This is to enforce that FROGBLOCK is not a public domain font and prevent it from ending up on free font hosting websites.

For more details, you may see the font and its license here: https://polyducks.itch.io/frogblock

Many thanks.

Ben Jones / Polyducks

**Comment by slmjkdbtl** [comment]
Hi! Sorry for the misuse, absolutely happy to include credit in the file name 336f3b6. FYI we're also using your [Kitchen Sink font](https://github.com/replit/kaboom/blob/master/src/assets/sink.png), and one [modified version with outline](https://github.com/replit/kaboom/blob/master/src/assets/sink.png). But I see putting them on Github is stretching the license, I'll remove the font files on Github shortly and put a link for people to access your fonts themselves.

Love your fonts and your works btw, they're really cool!

**Comment by Polyducks** [comment]
Thank you, I really appreciate it! I'm glad you're enjoying the fonts and getting good mileage from them :~)

Kitchen Sink has a slightly different requirement in that it was also made by Retroshark. Please credit as "Polyducks and Retroshark". 

Thanks again!
